doc/networks.md

```diff
@@ -409,3 +409,22 @@ ipv6.routes                     | string    | ipv6 address          | -
 ipv6.routes.anycast             | boolean   | ipv6 address          | false                     | Allow the overlapping routes to be used on multiple networks/NIC at the same time.
 dns.nameservers                 | string    | standard mode         | -                         | List of DNS server IPs on physical network
 ovn.ingress\_mode               | string    | standard mode         | l2proxy                   | Sets the method that OVN NIC external IPs will be advertised on uplink network. Either `l2proxy` (proxy ARP/NDP) or `routed`.
+
+# BGP integration
```

Comment:
This feels a bit redundant as the BGP config keys are already listed above for those network types.
I've added a mention in the config keys for physical that those are for use by downstream ovn networks though so that may help a bit.

Reply:
Thanks. I was just thinking it helps the reader who's reading about a particular network driver type to discover the functionality that is supported by that network driver (who may not even know that LXD has BGP support for example).
This way the user doesn't have to know that BGP exists or have to read the entire document to discover it.

I've been doing something similar with the network forward functionality, cross linking between the network driver info sections into the network-forwards.md file.